test/test-pages/mozilla-1/expected.html

```diff
@@ -2,65 +2,76 @@
-                    <p><img src="http://mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/designed-redesigned-high-res.6efd60766484.png" data-processed="false" data-src="//mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/designed-redesigned.fbd3ee9402e6.png" data-high-res="true" data-high-res-src="//mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/designed-redesigned-high-res.6efd60766484.png" alt="" id="designed-mobile" /> </p>
+                    <p><img src="http://mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/designed-redesigned.fbd3ee9402e6.png" alt="" id="designed-mobile" />
+                    </p>
                 </div>
                 <div id="flexible-bottom-animation">
-                    <p><img src="http://mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/animations/flexible-bottom-fallback.cafd48a3d0a4.png" alt="" /> </p>
+                    <p><img src="http://mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/animations/flexible-bottom-fallback.cafd48a3d0a4.png" alt="" /></p>
                 </div>
             </div>
         </section>
-        <section id="customize" data-ga-label="More ways to customize"> </section>
+        <section id="customize" data-ga-label="More ways to customize">
+        </section>
         <div id="customizers-wrapper">
             <section id="themes" role="tabpanel" aria-labelledby="customize-themes">
                 <div>
                     <div>
                         <h3>Themes</h3>
                         <p>Make Firefox match your style. Choose from thousands of themes and dress up your browser with a single click.</p>
-                        <p><a rel="external" href="https://addons.mozilla.org/firefox/themes/">Try it now</a> <br/> <a rel="external" href="https://support.mozilla.org/kb/use-themes-change-look-of-firefox">Learn more</a> </p>
+                        <p><a rel="external" href="https://addons.mozilla.org/firefox/themes/">Try it now</a>
+                            <br /> <a rel="external" href="https://support.mozilla.org/kb/use-themes-change-look-of-firefox">Learn more</a>
+                        </p>
                     </div>
-                    <p><a href="#add-ons" role="button">Next</a> </p>
-                    <p><img id="theme-demo" src="http://mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/theme-red.61611c5734ab.png" alt="Preview of the currently selected theme" /> </p>
+                    <p><a href="#add-ons" role="button">Next</a>
+                        <img id="theme-demo" src="http://mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/theme-red.61611c5734ab.png" alt="Preview of the currently selected theme" />
```

Comment:
This image used to be in its own `<p>` and now it's inline, which is probably not the right thing in the circumstances. Can we avoid this somehow? Perhaps we can check if the div/p which contains only the image and which we're removing has inline element or text siblings?

Reply:
It's fixed in [38bcb08](https://github.com/mozilla/readability/pull/588/commits/38bcb08e0c1c31f384d56c8b2c3098a07237dda1).